### PR TITLE
Fix/ooniprobe setup

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: customize [hunk 1]
```diff
@@ -31,17 +31,21 @@ set_apt_sources() {
     cat <<EOF >> ${ROOTDIR}/etc/apt/sources.list
 # Repository: $SUITE
 deb $APT_MIRROR $SUITE $COMPONENTS
-deb-src $APT_MIRROR $SUITE $COMPONENTS
```
Comment (bassosimone) [inline customize:34]: I don't understand the impact of removing this line. Are you removing support for debian-src repository?

File: lepidopter-fh/setup-ooniprobe.sh [hunk 2]
```diff
@@ -1,13 +1,35 @@
...
+TOR_REPO_GPG="A3C4F0F979CAA22CDBA8F512EE8CBC9E886DDD89"
+
+# Add Torproject Debian repository
+apt-key adv --keyserver hkp://pool.sks-keyservers.net --recv-keys ${TOR_REPO_GPG}
+echo "deb ${TOR_DEB_REPO} ${DEB_RELEASE} main" > ${TOR_DEB_REPO_SRC_LIST}
+
 # Install golang-go
 apt-get update
-apt-get -y -t stretch install golang-go
+apt-get -y -t ${DEB_RELEASE}-backports install golang-go
```
Comment (bassosimone) [inline lepidopter-fh/setup-ooniprobe.sh:14]: Here you install go from backports rather than from stretch. Why?

File: lepidopter-fh/setup-ooniprobe.sh [hunk 3]
```diff
@@ -1,13 +1,35 @@
...
+# Build meek-client
+export GOPATH=$(mktemp -d)
+go get git.torproject.org/pluggable-transports/meek.git/meek-client
+cp $GOPATH/bin/meek-client /usr/local/bin/meek-client
+chmod +x /usr/local/bin/meek-client
+rm -rf $GOPATH
+# Install ooniprobe and pluggable transports dependencies
+apt-get -y install openssl libssl-dev libyaml-dev libsqlite3-dev libffi-dev \
+libpcap0.8-dev libgeoip-dev libdumbnet-dev tor tor-geoipdb python-dev \
+libgmp-dev python-pip
```
Comment (bassosimone) [inline lepidopter-fh/setup-ooniprobe.sh]: Could you indent of some spaces the two lines above to enhance readability? Thanks!

File: lepidopter-fh/setup-ooniprobe.sh [hunk 4]
```diff
@@ -1,13 +1,35 @@
...
+# Install obfsproxy and fteproxy
+pip install obfsproxy fteproxy
+# Install obfs4proxy
+apt-get -y install -t stretch obfs4proxy
 # Install ooniprobe
-./scripts/install.sh
+# Remove previous version of pyasn1
+apt-get -y remove python-pyasn1
+pip install git+https://github.com/TheTorProject/ooni-probe.git
```
Comment (bassosimone) [inline lepidopter-fh/setup-ooniprobe.sh:34]: Cool, I didn't know this was possible! Is this pulling from master? Is this a thing we want? Isn't it more reproducable to install using a specific tag?

File: lepidopter-vmdebootstrap_build.sh [hunk 5]
```diff
@@ -12,8 +12,6 @@ APT_MIRROR="http://httpredir.debian.org/debian"
 MIRROR="http://httpredir.debian.org/debian"
 TODAY=`date +%Y%m%d-%H%M`
 
-export APT_MIRROR
```
Comment (bassosimone) [inline lepidopter-vmdebootstrap_build.sh:15]: Why was this variable removed? Is this related to the addition of `-a`?

File: customize (hunk 1 above)
Comment (anadahz) [inline customize:34]: The deb-src is a repository that include the source packages needed to build or compile packages. I don't think that this will be need in lepidopter.

File: lepidopter-fh/setup-ooniprobe.sh (hunk 2 above)
Comment (anadahz) [inline lepidopter-fh/setup-ooniprobe.sh:14]: jessie-backport repo [version](https://packages.debian.org/jessie-backports/golang-go): 2:1.5.1-4~bpo8+1
stretch repo [version](https://packages.debian.org/stretch/golang-go): 2:1.5.1-4
I don't think that there is a version difference (maybe minor differences), and installing a package from testing (stretch) with many dependencies is not really nice for the rest of the package dependencies in case of upgrades.

File: customize (hunk 1 above)
Comment (bassosimone) [inline customize:34]: ack!

File: lepidopter-fh/setup-ooniprobe.sh (hunk 2 above)
Comment (bassosimone) [inline lepidopter-fh/setup-ooniprobe.sh:14]: After a chat with @anadahz I got it :+1:

File: lepidopter-fh/setup-ooniprobe.sh (hunk 3 above)
Comment (anadahz) [inline lepidopter-fh/setup-ooniprobe.sh]: Thanks i 'll add some indentation for better readability.

File: lepidopter-fh/setup-ooniprobe.sh (hunk 4 above)
Comment (anadahz) [inline lepidopter-fh/setup-ooniprobe.sh:34]: Lepidopter release needs the cutting-edge ooniprobe release --this is
actually the whole point of this branch. For instance the [pypi version](https://pypi.python.org/pypi/ooniprobe) of ooniprobe 1.3.2 is missing [annotations](https://github.com/TheTorProject/ooni-probe/commit/6837a3ccccb01aea0381defa20847001c145f3a1), the OpenVPN [install script](https://github.com/TheTorProject/ooni-probe/commit/dbfaf075481c84f448485b8940b0676961cec81f), the [fix](https://github.com/TheTorProject/ooni-probe/commit/b792e1e340ba3c318f44b7177cec489092b63e26) to support twisted version >=15.5, and a crucial [fix](%28https://github.com/TheTorProject/ooni-probe/commit/592935063718b0836bd04ba6173fb58ada8b2ac5%29) for the lantern test.

File: lepidopter-vmdebootstrap_build.sh (hunk 5 above)
Comment (anadahz) [inline lepidopter-vmdebootstrap_build.sh:15]: Correct. The parameter `-a` of set command marks variables which are modified or created for export.

**Comment by anadahz** [comment]
Thank you @bassosimone for reviewing this PR!

File: lepidopter-fh/setup-ooniprobe.sh (hunk 4 above)
Comment (bassosimone) [inline lepidopter-fh/setup-ooniprobe.sh:34]: I understand, thanks!

File: lepidopter-vmdebootstrap_build.sh (hunk 5 above)
Comment (bassosimone) [inline lepidopter-vmdebootstrap_build.sh:15]: ack, thanks!
